docs/source/conf.py

```diff
@@ -49,7 +49,10 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo', 'sphinx.ext.coverage', 'sphinx.ext.imgmath', 'sphinx.ext.ifconfig', 'sphinx.ext.viewcode', 'IPython.sphinxext.ipython_console_highlighting', 'IPython.sphinxext.ipython_directive', 'sphinxcontrib.contentui', 'aiida.sphinxext']
+extensions = ['sphinx.ext.intersphinx', 'sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo',
+              'sphinx.ext.coverage', 'sphinx.ext.imgmath', 'sphinx.ext.ifconfig', 'sphinx.ext.viewcode',
+              'IPython.sphinxext.ipython_console_highlighting', 'IPython.sphinxext.ipython_directive',
+              'sphinxcontrib.contentui', 'aiida.sphinxext']
 ipython_mplbackend = ''
```

Comment:
I see some of the changes that I think you wanted to address in #3876 have snuck in. Maybe it is best to remove these here and only apply this in #3876

Reply:
Well, not quite - I added some new references to `flask`, which then caused the docs build to fail, which then caused me to add an intersphinx pointer to the flask docs here.
Which then caused me to think about adding also the docs of the python standard library (which is #3876)

Reply:
But then don't we just want to first merge #3876 which should be fine without this PR and then you rebase _this_ PR on top of that, which then only includes relevant changes, no?

Reply:
well, it's just extra effort and I think it's ok to keep the flask intersphinx mapping in this PR - otherwise I need to add another nitpick here and remove it again.
if you want me to do this, let me know.

Reply:
I played around a bit with the intersphinx and nitpicks, and rebased your other PR and merged it. This PR is now also rebased and you already addressed my comments, so will also merge this. Thanks a lot!